### PR TITLE
Xpanse alert update comment

### PR DESCRIPTION
**Pull request by edibleShell**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added the resolution_comment argument to the asm-update-alerts command. This will allow users to leverage the comment parameter when interacting with the /public_api/v1/alerts/update_alerts/ API endpoint. 

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@edibleShell thank you for your contribution.  I am looking into the underlying [API](https://cortex-panw.stoplight.io/docs/cortex-xpanse/6793ed71c04ac-update-alerts) and seeing if it should update resolution comment, if so we are good and I will work on updating API documentation to be more specific.

@aaron1535 please hold for now

**Review by johnnywilkes — APPROVED** [review]
@edibleShell , thanks for the contribution

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

@aaron1535 , this is approved by Xpanse, please merge when possible

File: Packs/CortexXpanse/ReleaseNotes/1_2_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Cortex Xpanse
+
+- Added the resolution_comment argument to the asm-update-alerts command
+- Updated asm-update-alerts documentation to include this new argument
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_2_4.md:7]:
```suggestion
- Updated the **asm-update-alerts** documentation to include this new argument.
```

**Comment by ShirleyDenkberg** [comment]
@johnnywilkes @aaron1535 Doc review completed.

**Review by aaron1535 — APPROVED** [review]

**Pull request merged**